liiweb/views/legislation.py

```diff
@@ -89,7 +89,7 @@ def add_children(self, queryset):
         children = defaultdict(list)
         children_qs = Legislation.objects.filter(
             parent_work_id__in=parents, repealed=False, metadata_json__principal=True
-        )
+        ).order_by("-date")
```

Comment:
This is interesting... this only impacts the legislation views which are by default sorted by title. So maybe we leave this out here?

Basically, we don't currently sort the children based on what the main listing is sorting by.

Reply:
@longhotsummer I think the request was that the children be sorted according to latest first,

Reply:
ok, let's do this globally then. All good.